--- origsrc/libaccounts-glib-1.4/libaccounts-glib/Makefile.am	2012-10-18 01:58:02.000000000 -0500
+++ src/libaccounts-glib-1.4/libaccounts-glib/Makefile.am	2012-12-29 21:04:37.390701400 -0600
@@ -13,6 +13,7 @@ libaccounts_glib_la_CFLAGS = \
 libaccounts_glib_la_LIBADD = $(LIBACCOUNTS_LIBS) $(AEGISCRYPTO_LIBS) -lrt
 libaccounts_glib_la_LDFLAGS = \
 	-version-info 1:3:1 \
+	-no-undefined \
 	-export-symbols-regex '^ag_'
 
 BUILT_SOURCES = \
